shared/src/commonMain/kotlin/co/touchlab/kampstarter/models/BreedModel.kt

```diff
@@ -66,12 +68,11 @@ class BreedModel(
         }
     }
 
-    private fun insertBreedData(breeds: List<String>) = mainScope.launch {
-        dbHelper.insertBreeds(breeds)
-    }
-
-    fun updateBreedFavorite(breed: Breed) = mainScope.launch {
+    fun updateBreedFavorite(breed: Breed) = scope.launch {
         dbHelper.updateFavorite(breed.id, breed.favorite != 1L)
+
+        // TODO tmp make an extra network call after a db update
+        ktorApi.getJsonFromApi()
```

Comment:
This is how I was making sure things still work, but we should remove before merging